assets/js/userCtrl.js

```diff
@@ -1,5 +1,9 @@
 /* global io, define */
-define(['lodash', 'regex'], function (_, regex) {
+define(['lodash', 'regex', 'sails', 'socket'], function (_, regex, sails, socket) {
+
+  var io = sails(socket);
+
+  io.sails.url = 'http://localhost:1337';
```

Comment:
This should probably be in a config file, since the url will vary between dev and production servers.